__tests__/resolve-config.test.ts

```diff
@@ -85,4 +85,24 @@ describe('Resolves Config', () => {
     process.env = {}
     await expect(resolveConfig.resolveConfig(mockReporter)).rejects.toThrowError()
   })
+
+  describe('parses boolean', () => {
+    test('but throws if incorrect value', async () => {
```

Comment:
It looks like the error is [not caught](https://github.com/DataDog/synthetics-ci-github-action/blob/4a7ded146c22543258f84d8259a1a414d524941d/src/main.ts#L17) in the `main.ts#run` function. Does that mean we would end up with an "Unhandled rejection" error at the GitHub Action level?

I wonder if we should do something like the following, or if it's overkill/useless to do so... 🤔 

```ts
// main.ts :: run

let config: synthetics.CommandConfig
try {
  config = await resolveConfig(reporter)
} catch(error) {
  core.setFailed(error)
  return
}
```

(said otherwise, is it okay to have an unhandled rejection, or do we _have_ to handle it?)

Reply:
Good point.

There're a couple of places (like for example [here](https://github.com/DataDog/synthetics-ci-github-action/blob/andrii.kalinichenko/SW-2510/add-tunnel-var/src/resolve-config.ts#L34)) triggers the error which is kind of `unhandled` in the end.

aligned my change with it, setting the exit code at once.